maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -265,7 +267,8 @@ data class TapOnPointV2Command(
     val point: String,
     val retryIfNoChange: Boolean? = null,
     val longPress: Boolean? = null,
-    val repeat: TapRepeat? = null
+    val repeat: TapRepeat? = null,
+    val waitToSettleTimeoutMs: Int? = null
```

Comment:
Maybe. Although it seems we're kinda arbitrarily apply.

Reply:
Kinda torn on this:
1. Most command params don't eval JS, unless users ask for it
2. IMO JS eval should not happen here as it will force all models to use string params (only)

Let's leave it as it is and reconsider if there's demand for it.